### PR TITLE
Added blocks that can be washed away by water

### PR DESCRIPTION
**Pull request by Bond-009**
http://minecraft.gamepedia.com/Breaking#Best_tools

Breaking double tall grass with water gives wrong drops (it should drop nothing)

### CODE REVIEW

**Comment by Seadragon91** [comment]
In the link you posted, I see that all rails can be washed away.

**Comment by Bond-009** [comment]
Fixed

**Comment by Seadragon91** [comment]
What means `washed away`? Block is destroyed, when water is placed on it or it can be transported by water?

File: src/Simulator/FluidSimulator.cpp [hunk 1]
```diff
@@ -31,26 +31,36 @@ bool cFluidSimulator::CanWashAway(BLOCKTYPE a_BlockType)
...
+	case E_BLOCK_RED_MUSHROOM:
+	case E_BLOCK_RED_ROSE:
+	case E_BLOCK_SNOW:
+	case E_BLOCK_SUGARCANE:
+	case E_BLOCK_TALL_GRASS:
+	case E_BLOCK_TORCH:
+	case E_BLOCK_TRIPWIRE_HOOK:
+	case E_BLOCK_TRIPWIRE:
+	case E_BLOCK_YELLOW_FLOWER:
+	{
```
Comment (Seadragon91) [inline src/Simulator/FluidSimulator.cpp]: Sorry, please correct the format.

Comment (Bond-009) [inline src/Simulator/FluidSimulator.cpp]: I already did

**Pull request merged**